src/glean.js

```diff
@@ -20,7 +20,7 @@ class Glean {
       appId = browser.runtime.id;
     } else {
       console.log("Running from a web page");
-      let gleanScript = document.querySelector('[src*=glean\\.js]');
+      let gleanScript = document.getElementById("glean-js");
```

Comment:
We could change the data attribute used in the Glean script to `data-glean-app-id` and here use `document.querySelector('[data-glean-app-id]')`. No need to change this here, just a thought.

Reply:
Oh, yes, that's probably a good idea. :) Let's keep this in our mind for post-this-week!